client/src/elm/Pages/WellChild/Activity/Utils.elm

```diff
@@ -751,6 +768,25 @@ dangerSignsTasksCompletedFromTotal currentDate assembled data task =
     resolveTasksCompletedFromTotal tasks
 
 
+mandatoryDangerSignsTasksCompleted : NominalDate -> Site -> AssembledData -> Bool
+mandatoryDangerSignsTasksCompleted currentDate site assembled =
+    resolvedMandatoryDangerSignsTasksCompleted site assembled
+        |> List.all (dangerSignsTaskCompleted currentDate assembled)
+
+
+resolvedMandatoryDangerSignsTasksCompleted : Site -> AssembledData -> List DangerSignsTask
+resolvedMandatoryDangerSignsTasksCompleted site assembled =
+    case ( site, assembled.encounter.encounterType ) of
```

Comment:
This is unreadable. Make it explicit.
For site Burundi => logic
For others => logic